cmd/maya-apiserver/spc-actions/select_disk.go

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
Can this is part of the spc-watcher itself? There is no need for a new spc-actions package.

Reply:
Will create a backlog for it

Reply:
https://rally1.rallydev.com/#/219541625440d/detail/userstory/247232707548

Reply:
Not sure about the reworks... But yes there will be some rework...
I can do it immediately but taking this as a separate PR would be good.